languages/Dockerfile.runner.j2

```diff
@@ -25,6 +25,9 @@ RUN /tmp/{{package.script}}
 {% endfor %}
 {% endfor %}
 
+RUN mkdir /opt/algorithm
+RUN chown $ALGO_UID:$ALGO_UID -R /opt/algorithm
+# Needed for run operations that download/use files in pwd
```

Comment:
What commands are those?  like above I'm not really opposed to this but I don't see anything doing that?  if algorithms need space they should create tmp folders and not use the working directory anyway.

Reply:
This is required for the `dotnet build` operation to function properly, as `algo` needs to own that location to do so.

Reply:
There isn't an easy way to tell the `dotnet` binary to use a different location for build products, it's all kind of hardcoded in unfortunately

Reply:
this is in the runner image which will never do a `dotnet build`

Reply:
`dotnet run` I believe requires this still

Reply:
keeping for now, may change in QoL before final release